### PR TITLE
Network check added in search fragment

### PR DESCRIPTION
**Pull request by rob729**
@sridharjajoo This PR is for issue #22.
Please review it.

### CODE REVIEW

File: app/src/main/java/com/example/sridharjajoo/newsapp/core/Search/SearchFragment.java [hunk 1]
```diff
@@ -117,7 +117,12 @@ public boolean onMenuItemActionCollapse(MenuItem item) {
         searchView.setOnQueryTextListener(new SearchView.OnQueryTextListener() {
             @Override
             public boolean onQueryTextSubmit(String query) {
-                handleSearch(query);
+                if(Utils.hasNetwork()) {
```
Comment (sridharjajoo) [inline app/src/main/java/com/example/sridharjajoo/newsapp/core/Search/SearchFragment.java]: space between if and '('

File: app/src/main/java/com/example/sridharjajoo/newsapp/core/Search/SearchFragment.java [hunk 2]
```diff
@@ -117,7 +117,12 @@ public boolean onMenuItemActionCollapse(MenuItem item) {
...
+                    handleSearch(query);
+                }
+                else {
```
Comment (sridharjajoo) [inline app/src/main/java/com/example/sridharjajoo/newsapp/core/Search/SearchFragment.java]: move else to previous line. Something like this } else {

File: app/src/main/java/com/example/sridharjajoo/newsapp/core/Search/SearchFragment.java [hunk 3]
```diff
@@ -117,7 +117,12 @@ public boolean onMenuItemActionCollapse(MenuItem item) {
...
+                    Toast.makeText(getActivity(),"Network not available!",Toast.LENGTH_SHORT).show();
```
Comment (sridharjajoo) [inline app/src/main/java/com/example/sridharjajoo/newsapp/core/Search/SearchFragment.java]: space between , and "Network not available" and , and Toast.LENGTH_SHORT

**Comment by rob729** [comment]
@sridharjajoo I have done the recommended changes.

**Pull request merged**